src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -1,104 +1,110 @@
+  --#{$notification-drawer}__header-title--FontSize: var(--pf-t--global--font--size--heading--md);
+  --#{$notification-drawer}__header-title--FontWeight: var(--pf-t--global--font--weight--heading--bold);
+  --#{$notification-drawer}__header-status--MarginLeft: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__header-status--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$notification-drawer}__header-status--FontSize: var(--pf-t--global--font--size--body--default);
+  --#{$notification-drawer}__header-status--Color: var(--pf-t--global--text--color--subtle);
 
   // Body
-  --#{$notification-drawer}__body--ZIndex: var(--#{$pf-global}--ZIndex--xs);
+  --#{$notification-drawer}__body--ZIndex: var(--pf-t--global--Zindex--xs);
+  --#{$notification-drawer}__body--PaddingBlockStart: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__body--PaddingBlockEnd: var(--pf-t--global--spacer--md);
+
+  // List
+  --#{$notification-drawer}__list--Gap: var(--pf-t--global--spacer--sm);
+  --#{$notification-drawer}__list--PaddingInlineStart: var(--pf-t--global--spacer--lg);
+  --#{$notification-drawer}__list--PaddingInlineEnd: var(--pf-t--global--spacer--lg);
 
   // List item
-  --#{$notification-drawer}__list-item--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$notification-drawer}__list-item--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$notification-drawer}__list-item--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$notification-drawer}__list-item--PaddingLeft: var(--#{$pf-global}--spacer--md);
-  --#{$notification-drawer}__list-item--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-  --#{$notification-drawer}__list-item--BoxShadow: inset var(--#{$pf-global}--BoxShadow--sm-bottom);
-  --#{$notification-drawer}__list-item--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$notification-drawer}__list-item--BorderBottomColor: transparent;
-  --#{$notification-drawer}__list-item--OutlineOffset: #{pf-size-prem(-4px)};
-  --#{$notification-drawer}__list-item--before--Width: var(--#{$pf-global}--BorderWidth--lg);
-  --#{$notification-drawer}__list-item--before--Top: 0;
-  --#{$notification-drawer}__list-item--before--Bottom: calc(var(--#{$notification-drawer}__list-item--BorderBottomWidth) * -1);
+  --#{$notification-drawer}__list-item--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__list-item--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__list-item--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__list-item--PaddingLeft: var(--pf-t--global--spacer--md);
+  --#{$notification-drawer}__list-item--BackgroundColor: var(--pf-t--global--background--color--primary--default);
+  --#{$notification-drawer}__list-item--BorderWidth: var(--pf-t--global--border--width--box--status--default);
+  --#{$notification-drawer}__list-item--BorderBottomColor: var(--pf-t--global--border--color--default);
```

Comment:
It dfoesn't seem like we still need this var? It's now only being used on line ~198 in the pf-m-read block, but we shouldn't need it since we're using the --pf-v5-c-notification-drawer__list-item--BorderColor var